sarahricker/thinkery
# Tessitura templates

## Description

Adds Tessitura template files. Tessitura will pull these files from production to use on my.thinkeryaustin.org.

## Steps to Test or Reproduce

1. Temporarily change all `https://thinkeryaustin.org` to `https://thinkery.wpengine.com`
2. You should be able to view the index.html files as real webpages.
3. Javascript will be broken. We don't include jQuery on purpose. JS won't work until this is used by Tessitura.

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/jPAdK8Nfzzwt2/giphy.gif)
